project_app/pages/Main.js

```diff
@@ -0,0 +1,94 @@
+import Button from "@mui/material/Button";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Grid from "@mui/material/Grid";
+import Container from "@mui/material/Container";
+import styled from "styled-components";
+import Image from "next/image";
+
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+
+const cards = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16];
+
+const theme = createTheme();
+
+export default function Main() {
+  return (
+    <ThemeProvider theme={theme}>
+      <Wrapper>
+        <Container maxWidth="lg">
+          <Grid container spacing={8}>
+            {cards.map((card) => (
+              <Grid item key={card} xs={8} sm={6} md={3}>
+                <Card
+                  sx={{
+                    display: "flex",
```

Comment:
can we have the whole Card component in a separate component pls thanks

Reply:
yes will do

Reply:
Split code and have a separate Card component